data/WI/incentives.json

```diff
@@ -0,0 +1,410 @@
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 cash back for self-installed attic insulation and air sealing."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-7",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "smart_thermostat",
+    "program": "wi_focusOnEnergy_smartThermostatsRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 50,
+      "minimum": 50,
+      "maximum": 50
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$50 for Smart Thermostats. Available as instant discounts in the online marketplace or as rebates."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-8",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "pos_rebate"
+    ],
+    "item": "heat_pump_water_heater",
+    "program": "wi_focusOnEnergy_waterHeatingRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 300,
+      "minimum": 300
+    },
+    "owner_status": [
+      "homeowner"
+    ],
+    "short_description": {
+      "en": "Instant discounts for electric heat pump water heaters starting at $300 through a Trade Ally contractor."
+    },
+    "bonus_available": true,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-10",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "rooftop_solar_installation",
+    "program": "wi_focusOnEnergy_residentialCustomerSolarPVRebates",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 500,
+      "minimum": 500,
+      "maximum": 500
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "Rural residential customers can receive an additional $500 for installing a system."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-11",
+    "authority_type": "state",
+    "authority": "wi-focus-on-energy",
+    "payment_methods": [
+      "rebate"
+    ],
+    "item": "weatherization",
+    "program": "wi_focusOnEnergy_dIYResidentialInsulation&AirSealingRebate",
+    "amount": {
+      "type": "dollar_amount",
+      "number": 200,
+      "minimum": 200,
+      "maximum": 200
+    },
+    "owner_status": [
+      "homeowner",
+      "renter"
+    ],
+    "short_description": {
+      "en": "$200 cash back for qualified self-installed attic insulation and air sealing."
+    },
+    "start_date": 2023,
+    "end_date": 2023,
+    "low_income": "wi-focus-on-energy"
+  },
+  {
+    "id": "WI-21",
```

Comment:
If I'm understanding correctly, W21 and W22 seem like out of date duplicates of W23 and W24. Or at least, I can't find anywhere on that page where W21/22 exist.

If so, you can keep the 2023 ones, but do update the 2024 ones to have the correct start/end dates so it's more obvious that these are dupes.

Reply:
Huh, good catch! I'll remove W21 and W22